docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc

```diff
@@ -407,22 +376,6 @@ Setting this to `true` is not recommended.
 
 // =============================================================================
 
-// tag::elasticsearch-username[]
-|
-[id="env-{type}-elasticsearch-username"]
-`ELASTICSEARCH_USERNAME`
-
-| (string) The basic authentication username used to connect to {es}.
-This user needs the privileges required to publish events to {es}.
-
-// To do: link to required privileges
-
-*Default:* `elastic`
-
-// end::elasticsearch-username[]
-
-// =============================================================================
-
 // tag::es-username[]
```

Comment:
The `es-username` tag and `es-password` tag are not being removed as they are used by the standalone kubernetes config (i think for the agent to send data into es)